scale-tests/spark-batch-workload.json

```diff
@@ -0,0 +1,56 @@
+{
+  "id": "/spark-batch-workload",
+  "backoffFactor": 1.15,
+  "backoffSeconds": 1,
+  "cmd": "cd $MESOS_SANDBOX; git clone https://github.com/mesosphere/spark-build.git; cd spark-build/; pwd; git checkout $SPARK_BUILD_BRANCH; python3 -m venv test-env; pip3 install -r tests/requirements.txt; dcos cluster setup https://master.mesos --username=$DCOS_UID --password=$DCOS_PASSWORD --no-check; dcos package install spark --cli --yes; cd scale-tests/; export PYTHONPATH=../spark-testing:../testing; python3 scale_test.py $MESOS_SANDBOX/$DISPATCHERS_FILENAME $SUBMITS_PER_MIN",
+  "container": {
+    "type": "DOCKER",
+    "volumes": [],
+    "docker": {
+      "image": "mesosphere/spark-build:latest",
+      "forcePullImage": false,
+      "privileged": false,
+      "parameters": []
+    }
+  },
+  "cpus": 2,
+  "disk": 1024,
+  "env": {
+    "DCOS_PASSWORD": "deleteme",
```

Comment:
Is there a different way to handle passwords?

Reply:
Metronome doesn't support secrets, but Marathon does. I could store the password in the secret store for the Marathon service.